tests/bip/test_seed.py

```diff
@@ -0,0 +1,121 @@
+from unittest import TestCase
+from typing import List, Tuple
+# from ragger.bip.seed import Seed
+from ragger import calculate_public_key_and_chaincode, CurveChoice
+from bip_utils import Bip32KeyError
+
+# Disable yapf as it proposes unreadable test vectors
+# yapf: disable
+
+class TestSeed(TestCase):
+
+    def test_get_public_key_and_chaincode_error_curve(self):
+        with self.assertRaises(ValueError):
+            calculate_public_key_and_chaincode(0, path="m/0'/0'/0'/0/0")
+
+    def test_get_public_key_and_chaincode_error_unhardened_path(self):
+        with self.assertRaises(Bip32KeyError):
+            calculate_public_key_and_chaincode(curve=CurveChoice.Ed25519Slip,
+                                               path="m/44'/601/1430/20/0/0")
+        with self.assertRaises(Bip32KeyError):
+            calculate_public_key_and_chaincode(curve=CurveChoice.Ed25519Blake2bSlip,
+                                               path="m/44'/601/1430/20/0/0")
+
+    def test_get_public_key_and_chaincode_secp256k1(self):
+        secp256k1_test_values: List[Tuple[str, str, str]] = (
```

Comment:
Technically, this is a `Tuple[Tuple[str, str, str]]` but we don't really care, we just need iterables.